README.md

```diff
@@ -107,7 +121,8 @@ off-line validation that the SHA hashes are the properly published hashes.
 One can provide a `BUILD.bazel file snippet` that will be substituted for the auto-generated target
 implied by a maven artifact.  This is very useful for providing an annotation-processor-exporting
 alternative target.  The substitution is naive, so the string needs to be appropriate and any rules
-need to be correct, contain the right dependencies, etc. 
+need to be correct, contain the right dependencies, etc.  To aid that it's also possible to (on a
+per-package basis) substitute dependencies on a given fully-qualified bazel target for another. 
 
 
 A simple use-case would be to substitute a target name (e.g. "mockito-core" -> "mockito") for
```

Comment:
I'm super confused, since the first snippet is mockito-core, the later one is dagger.  Did you expand the file? lines 131-157 are the mockito example, then the dagger example begins on line 162.